README.md

```diff
@@ -105,25 +91,53 @@ This will run the entire test suite, both on server and client apps.
 
 Some suites need MongoDB configured in Replica Set mode to run properly. The provided Docker Compose file runs MongoDB in Replica Set mode and initializes the cluster automatically, if you are using your own mongo installation Refer to [MongoDB's documentation](https://www.mongodb.com/docs/manual/tutorial/deploy-replica-set/#initiate-the-replica-set) for more information.
 
-#### End to End (e2e)
+There are also Cypress components tests. It's recommended that Cypress tests are run with Chrome, or Chrome based browsers.
 
-For End-to-End testing, we have a full set of fixtures that test the overall functionality. Be advised that, for the time being, these tests are run ON THE SAME DATABASE as the default database (uwazi_developmet), so running these tests will DELETE any existing data and replace it with the testing fixtures. DO NOT RUN ON PRODUCTION ENVIRONMENTS!
+You can run individual tests with the Cypress UI:
 
-Running end to end tests requires a running Uwazi app.
+```
+$ yarn cypress
+```
 
-Running tests with Nightmare
+or you can run tests in headless mode
 
 ```
-$ yarn hot
+$ yarn cy-components --browser chrome
+```
+
+### End to End testing (e2e)
+
+Running end to end tests requires a running Uwazi app. For End-to-End testing, we have a full set of fixtures that test the overall functionality. **It's not advised to run these tests on production environments**, since an incorrectly configured run can have unwanted effects on the production database.
```

Comment:
```suggestion
Running end-to-end tests requires a running Uwazi APP. For End-to-end testing, we have a full set of fixtures that test the overall functionality. **It's not advised to run these tests on production environments**, since an incorrectly configured run can have unwanted effects on the production database.
```